### PR TITLE
Facebook update; Readme better example;Chrome option

### PR DESCRIPTION
**Pull request by italojs**
# README
## Easy documentation
I had dificult to understand what this project do, so i improve the readme to be more easy to know what this project do.

## Incomplete example
In the JSON request example don't had the `type` fields, it's throw an error when the Facebook strategy will decide between group or feed post type

# Code
## Facebook page out of date
I guess the Facebook update the facebook group post HTML because don't exist an `button.touchable` element or attribute(of some element) and it was throwing an exeption, so i update the element search by `div._4g34._6ber._5i2i._52we` (same of the FEED)

## Chrome option
The chromium that puppeteer install by default don't was working in my machine, so I need install some google chrome and set the binary path to puppeteer, so I added this option in the AbstractInterface.js file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,12 @@
 
 ---
 
-The easiest way to spam social networks. Just create a strategy or use one of the existing ones.
+Welcome to the easiest way to spam social networks. <br>
```
Comment (IgorHalfeld) [inline README.md]: On markdown, two `spaces` make break line :D
Can you change the `<br>` to two spaces?

File: README.md [hunk 2]
```diff
@@ -22,15 +27,30 @@ The easiest way to spam social networks. Just create a strategy or use one of th
...
-  ...
+	"strategy":"Facebook",
+	"user": "<YOUR_EMAIL>",
+	"pass": "<YOUR_PASSWORD>",
+	"data": [
+		{ "id": "<group_id>",
+		  "text": "<Message>",
+		  "type": "GROUP or FEED"
+    },
+    { "id": "<group_id>",
```
Comment (IgorHalfeld) [inline README.md]: Can you fix this indentation?

File: lib/AbstractInterface.js [hunk 3]
```diff
@@ -1,7 +1,11 @@
 const puppeteer = require('puppeteer')
-const Plugin = require('../lib/plugins')
 
-const launchOptions = { args: ['--no-sandbox', '--disable-setuid-sandbox'] }
+const launchOptions = { 
+  args: ['--no-sandbox', '--disable-setuid-sandbox'] 
+}
+
+if (process.env.PUPPETEER_EXECUTABLE_PATH)
```
Comment (IgorHalfeld) [inline lib/AbstractInterface.js:7]: This is really nice haha

**Review by IgorHalfeld — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (IgorHalfeld) [inline README.md]: image

Still without correct indentation haha

**Review by IgorHalfeld — CHANGES_REQUESTED** [review]
Can you fix the README?

File: README.md (hunk 2 above)
Comment (italojs) [reply README.md]: WTF? in my vs code it's perfect.

Comment (italojs) [reply README.md]: ok, it was an bizarre blank character that include a big space in the line

**Review by IgorHalfeld — APPROVED** [review]

**Comment by IgorHalfeld** [comment]
Merged!

**Pull request merged**